install/infra/single-cluster/aws/main.tf

```diff
@@ -0,0 +1,6 @@
+terraform {
+  backend "s3" {
+    bucket = "gitpod-tf"
```

Comment:
**Question:** If a user directly uses this module, will this backend definition conflict with their environment?

Reply:
This is a good point! I could leave the bucket field empty since I cannot use variables in bucket definition. But I am also thinking, the chances of them having a bucket named `gitpod-tf` is unlikely, so it will error out saying no such bucket is present. Not sure what the right route is, I might leave this as is.